### ISSUE TITLE
AppIndicator Support (StatusNotifier)

### ISSUE THREAD

**Issue by ikeydoherty** [solus-project/budgie-desktop#286]
Budgie should support AppIndicator as it's primary "status tray"

 - For X11 users, keep legacy NaTray as an option
 - For Wayland, add `X-Display-Server` option to .plugin files to blacklist X11-specifics
 - Default to new AppIndicator/Status Notifier implementation

**Comment by koesherbacon** [comment]
I agree, especially for apps like Dropbox, Deluge, Transmission; you know the ones we all leave running with an app-indicator so our taskbar/dock/desktop isn't too cluttered.

**Comment by rilian-la-te** [comment]
I wrote experimental Budgie support for my implementation: https://github.com/rilian-la-te/xfce4-sntray-plugin/commit/8d4c7c3a6c3f1d921e18a1e386fa3dd1b3759fad

**Comment by rilian-la-te** [comment]
What do you think about it? This is functional, but not Budgie-integrated.

**Comment by eprincen2** [comment]
Is there any motion on this? Budgie is my favorite environment and this is a definite shortcoming. How can I help make this happen?
